lib/StencilCLISettings.js

```diff
@@ -0,0 +1,18 @@
+class StencilCLISettings {
+    constructor() {
+        // The setting enables/disables verbose network requests logging
+        this.versboseNetworkLogging = true;
+    }
+
+    setVerbose(flag) {
+        this.versboseNetworkLogging = flag;
+    }
+
+    isVerbose() {
+        return this.versboseNetworkLogging === true;
```

Comment:
Cannot we just do as follows?
```suggestion
        return this.versboseNetworkLogging;
```

Reply:
@bc-max yeah, we can